app/logic/transcript.py

```diff
@@ -36,7 +36,8 @@ def getProgramTranscript(username):
     EventData = (Event
         .select(Event, fn.SUM(EventParticipant.hoursEarned).alias("hoursEarned"))
         .join(EventParticipant)
-        .where(EventParticipant.user == username)
+        .where(EventParticipant.user == username, 
+        Event.program != 9)
```

Comment:
The where clause is short enough to all be on the same line, also we should not be using a magic number.